mods/wlssb/statuses.js

```diff
@@ -186,6 +186,7 @@ exports.BattleStatuses = {
 	lycaniumz: {
 		exists: true,
 		onStart: function (pokemon) {
+			pokemon.types = ["Dark"];
```

Comment:
"Why is this here?" You ask? Well apparently the aesthetics say Arceus-Dark but i found out that its actually arceus-normal. This makes sure its actually arceus-dark through and through.